blocks/index.js

```diff
@@ -14,6 +14,7 @@ import './hooks';
 // and then stored as objects in state, from which it is then rendered for editing.
 export * from './api';
 export * from './autocompleters';
+export * from './inline-blocks';
```

Comment:
Currently we have a `core-blocks` directory to the top level? Would it make sense to have this one at the top level too with a similar way to register inline blocks?

Reply:
I like that idea! Commit https://github.com/WordPress/gutenberg/pull/5794/commits/4403d200433ded292223d7c9bb849b285eabbce5 sets up `core-inlineblocks` and `inline-blocks` directories to mirror the `core-blocks` and `blocks` directories and adds a similar way to register inline blocks. It's a work in progress.